site/quickstart.md

```diff
@@ -249,6 +249,11 @@ $ curl -v -H 'host: sample-domain1.org' http://your.server.com:30305/weblogic/
 **Note**: Depending on where your Kubernetes cluster is running, you may need to open firewall ports or
 update security lists to allow ingress to this port.
 
+f.	To access the WLS Administration Console, uncomment the `NodePort` setting in your `domain.yaml` file and open a browser to http://your.server.com:30305. Substitute the Node IP address of the worker node for `your.server.com`. You can find it by running:
```

Comment:
in the Quick start guide sample the domain.yaml is generated by entries in the template  `kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml`.  There are instructions before to copy the sample `kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml` and a note For example, assuming you named your copy `my-inputs.yaml`.

Reply:
I would say 
To access the WLS Administration Console, uncomment the `NodePort` setting in my-inputs.yaml (assuming you named your copy my-inputs.yaml) file and open a browser to http://your.server.com:30305.

Reply:
Please review these changes to make sure that I got in the information you wanted. Thanks.